packages/main/src/themes/CustomListItem.css

```diff
@@ -10,8 +10,9 @@
 }
 
 .ui5-li-root.ui5-custom-li-root {
-	padding: 0;
 	pointer-events: inherit;
+	min-height: var(--_ui5_list_item_base_height);
```

Comment:
This will lock the `min-height` and an override on the host would not work. Perhaps a value `inherit` will also work?

Also, doesn't this work without the `overflow: hidden;`? I'm OK to keep it if it's really needed.